scss/docs/_patterns_icons.scss

```diff
@@ -41,3 +41,5 @@
     margin-right: $sph-inner;
   }
 }
+
+@include vf-p-icons--application;
```

Comment:
Nice, good place for those! Just add a comment referring to where they are needed in docs, thx!